mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -252,3 +252,38 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 		ARG 1 sender
 	METHOD method_29043 isMultiplayerEnabled ()Z
 	METHOD method_29044 isOnlineChatEnabled ()Z
+	METHOD method_29599 createSaveProperties (Lnet/minecraft/class_32$class_5143;Lnet/minecraft/class_5318$class_5319;Lnet/minecraft/class_3300;Lnet/minecraft/class_5359;)Lnet/minecraft/class_5219;
+		ARG 0 session
+		ARG 1 dimensionTracker
+		ARG 2 resourceManager
+	METHOD method_29603 (Lnet/minecraft/class_5318$class_5319;Lnet/minecraft/class_32$class_5143;Lnet/minecraft/class_310$class_5367;Lnet/minecraft/class_5219;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnet/minecraft/class_3312;Ljava/lang/Thread;)Lnet/minecraft/class_1132;
+		ARG 1 dimensionTracker
+		ARG 2 session
+		ARG 4 saveProperties
+		ARG 5 sessionService
+		ARG 6 profileRepository
+		ARG 7 userCache
+		ARG 8 serverThread
+	METHOD method_29606 startIntegratedServer (Ljava/lang/String;)V
+		ARG 1 worldName
+	METHOD method_29607 (Ljava/lang/String;Lnet/minecraft/class_1940;Lnet/minecraft/class_5318$class_5319;Lnet/minecraft/class_5285;)V
+		ARG 1 worldName
+		ARG 2 levelInfo
+		ARG 3 dimensionTracker
+		ARG 4 generatorOptions
+	METHOD method_29610 startIntegratedServer (Ljava/lang/String;Lnet/minecraft/class_5318$class_5319;Ljava/util/function/Function;Lcom/mojang/datafixers/util/Function4;ZLnet/minecraft/class_310$class_5366;)V
+		ARG 1 worldName
+		ARG 2 dimensionTracker
+		ARG 5 safemode
+	CLASS class_5366 WorldLoadAction
+	CLASS class_5367
```

Comment:
Might name this thing

Reply:
I can't exactly figure out what it means. It seems to be a combo of ServerResourceManager, resource pack manager and another thing.

IntegratedResourceManager was my first thought